tests/unit/querybuf.tcl

```diff
@@ -92,5 +138,23 @@ start_server {tags {"querybuf slow"}} {
      
         $rd close
     }
+}
+
+start_server {tags {"querybuf"}} {
+    test "Client executes small argv commands using shared query buffer" {
+        set rd [redis_deferring_client]
+        $rd client setname test_client
+        set res [r client list]
```

Comment:
we have no guarantee that the previous command (setname) was run (missing `rd read`)

Reply:
missing `rd read`, fixed it.